### PR TITLE
Chore: Add user state util functions

### PR DESCRIPTION
**Pull request by GaboDimitrov**
Chore: Add user state util functions in util package

### CODE REVIEW

File: packages/utils/__tests__/user-state.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { isLoggedIn, isMeteredExpired, isShared, isMetered } from '../src';
+
+const defaultUserState = {
+    isLoggedIn: true,
+    isMetered: false,
+    isMeteredExpired: false,
+    isShared: true
+};
+
+describe.only('user state should', () => {
```
Comment (ValGeorgiev) [inline packages/utils/__tests__/user-state.test.js]: why you need only here?

File: packages/utils/__tests__/user-state.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    it('return true if userState isLoggedIn is true', () => {
+        const userState = defaultUserState;
```
Comment (ValGeorgiev) [inline packages/utils/__tests__/user-state.test.js]: this is not needed here, you can pass defaultUserState straight away

File: packages/utils/__tests__/user-state.test.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        expect(isLoggedIn(userState)).toBe(true);
+    });
+
+    it('return true if user.isLoggedIn equals true and iuser.isMeteredExpired equals true', () => {
+        const userState = {...defaultUserState, isMeteredExpired: true};
+        expect(isMeteredExpired(userState)).toBe(true);
+    });
+
+    it('sreturn false if user.isLoggedIn equals true and iuser.isMeteredExpired equals false', () => {
```
Comment (ValGeorgiev) [inline packages/utils/__tests__/user-state.test.js]: typo

File: packages/utils/__tests__/user-state.test.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    isShared: true
+};
+
+describe.only('user state should', () => {
+    it('return true if userState isLoggedIn is true', () => {
+        const userState = defaultUserState;
+        expect(isLoggedIn(userState)).toBe(true);
+    });
+
+    it('return true if user.isLoggedIn equals true and iuser.isMeteredExpired equals true', () => {
```
Comment (ValGeorgiev) [inline packages/utils/__tests__/user-state.test.js]: iuser typo

File: packages/utils/__tests__/user-state.test.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
+import { isLoggedIn, isMeteredExpired, isShared, isMetered } from '../src';
+
+const defaultUserState = {
+    isLoggedIn: true,
+    isMetered: false,
+    isMeteredExpired: false,
+    isShared: true
```
Comment (ValGeorgiev) [inline packages/utils/__tests__/user-state.test.js]: lets isShared to be false in the default state

File: packages/utils/__tests__/user-state.test.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    const userState = { ...defaultUserState, isShared: true };
+    expect(isShared(userState)).toBe(true);
+  });
+
+  it("return true if user.isLoggedIn equals true and user.isMetered equals true", () => {
+    const userState = { ...defaultUserState, isMetered: true };
+    expect(isMetered(userState)).toBe(true);
+  });
+
+  it("return true if user.isLoggedIn equals true and user.isMetered equals true", () => {
```
Comment (nehasri89) [inline packages/utils/__tests__/user-state.test.js]: the test is checking for shared `userstate`, description needs to change

File: packages/utils/__tests__/user-state.test.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+  it("return false if user.isLoggedIn equals true and iuser.isMeteredExpired equals false", () => {
+    expect(isMeteredExpired(defaultUserState)).toBe(false);
+  });
+
+  it("return true if userState.isShared equals true", () => {
+    const userState = { ...defaultUserState, isShared: true };
+    expect(isShared(userState)).toBe(true);
+  });
+
+  it("return true if user.isLoggedIn equals true and user.isMetered equals true", () => {
```
Comment (nehasri89) [inline packages/utils/__tests__/user-state.test.js]: should have test if `user.isLoggedIn` is false

**Comment by ValGeorgiev** [comment]
The lint is failing in utils package, also could you change the PR name to `chore: Add user state util functions`

**Pull request merged**